AI/BattleAI/BattleAI.cpp

```diff
@@ -282,7 +282,7 @@ BattleAction CBattleAI::activeStack( const CStack * stack )
 BattleAction CBattleAI::goTowardsNearest(const CStack * stack, std::vector<BattleHex> hexes) const
 {
 	auto reachability = cb->getReachability(stack);
-	auto avHexes = cb->battleGetAvailableHexes(reachability, stack, true);
+	auto avHexes = cb->battleGetAvailableHexes(reachability, stack, false);

```

Comment:
Why battleAI changes are needed?

Reply:
I added a new parameter to getAvailableHexes, so that this function knows about hovered unit. And thi function is used in AI for whatever purposes.